splitio/CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 0.1.2+2
```

Comment:
Is not the date necessary?

Reply:
Probably a good idea to add them, thanks